subsys/settings/src/settings_line.c

```diff
@@ -226,7 +226,7 @@ static int settings_line_raw_read_until(off_t seek, char *out, size_t len_req,
 	size_t exp_size, read_size;
 	u8_t rbs = settings_io_cb.rwbs;
 	off_t off;
-	int rc;
+	int rc = 0;
```

Comment:
not really an issue as well, this function is never called for 0-length read-out.